examples/1d/plot_real_signal.py

```diff
@@ -120,7 +120,7 @@
 plt.subplot(3, 1, 3)
 plt.imshow(Sx[order2], aspect='auto')
 plt.title('Second-order scattering')
-
+plt.tight_layout()
```

Comment:
Could you move this to just above `plt.show()` (as you did for `plot_synthetic`)?